### ISSUE TITLE
unable to reset the QPs after setting an initial value

### ISSUE THREAD

**Issue by shankarsridhar** [knownasilya/ember-query-params#23]
#### TL;DR ###
Whats the recommended way to reset the QPs after setting them to some values on page load ?

-------------
In my controller, I do something like :
```
paramsRelay.setParam(paramName, undefined);
```
where,
`paramsRelay: Ember.inject.service()`
`paramName` is a string, say 'detected_via'.

But on debugging, I see that the paramName is first set to `undefined` and then to the string *'undefined'*. This `set`ting of undefined and then to string 'undefined', happens inside the `queryParamsDidChange` method inside the autosubscribe.js file.

Not sure if this is a bug or if I am setting the QPs the wrong way.

_What am I trying to do ?_
I have my QPs enabled on page load. Then I wanna remove the query Param when the checkbox is  turned off by user.

I would be happy to give more information, if needed.

**Comment by shankarsridhar** [comment]
bump ^^
